src/mapping/NearestProjectionMapping.hpp

```diff
@@ -49,8 +49,7 @@ class NearestProjectionMapping : public Mapping {
 private:
   logging::Logger _log{"mapping::NearestProjectionMapping"};
 
-  using InterpolationElements = std::vector<query::InterpolationElement>;
-  std::vector<InterpolationElements> _weights;
+  std::vector<InterpolationElement> _interpolationElements;
```

Comment:
Let's use simply `_interpolations` here. As described above, they are not "elements", but in this case they are all interpolations.